dist/common/scripts/scylla_prepare

```diff
@@ -75,9 +73,5 @@ if __name__ == '__main__':
             run('hugeadm --create-mounts')
         fi
     else:
-        set_nic_and_disks = get_set_nic_and_disks_config_value(cfg)
-        ifname = cfg.get('IFNAME')
```

Comment:
I'm not sure I understand why you choose to run `perftune.py` unconditionally.
What if the user answered "No" both to the question if he/she wants to tune a clocksource and about NIC and I/O tuning? I think that in this case we are still expected *not* to run a `perftune.py` like before.

Please, let me know if I missed something.

Reply:
We're not running perftune unconditionally. We're doing exactly as you described. However, the logic was moved to scylla-Util, where it looked more natural. The condition on scylla_prepare is "if create_perftune_conf(cfg): run("..."

Reply:
I thought I have deleted this comment. ;)
Yeah, I saw that further in the patch.

This also means that I have probably deleted some other comment I shouldn't have... :D